docs/getting-started/testing-solidus.mdx

```diff
@@ -135,7 +150,7 @@ module AmazingStore
 
         # Optional: Any factories you want to require from extensions.
         extension_paths = [
-          # MySolidusExtension::Engine.root.join("lib/solidus_content/factories/order.rb"),
+          # MySolidusExtension::Engine.root.join("lib/solidus_content/factories.rb"),
```

Comment:
🤔 I'm not sure about the current situation with extensions, but maybe the correct path is:

```rb
MySolidusExtension::Engine.root.join("lib/my_solidus_extension/factories"),
```

assuming no `factories.rb` is requiring subpaths, not having an extension should pickup a directory if existing and the `.rb` file otherwise.

Alternatively we could add an `# or` to make it clear that some extensions will have a file, while others will have a folder.

Reply:
Right, what about something like:

```
        extension_paths = [
          # MySolidusExtension::Engine.root.join("lib/my_solidus_extension/testing_support/factories"),
          # or individually:
          # MySolidusExtension::Engine.root.join("lib/my_solidus_extension/testing_support/factories/resource.rb"),
        ]
```

Clear enough?

Reply:
Absolutely! 👍👍👍